src/go/pt-galera-log-explainer/types/conflicts.go

```diff
@@ -0,0 +1,57 @@
+package types
+
+type Conflicts []*Conflict
+
+type Conflict struct {
+	Seqno       string
+	InitiatedBy []string
+	Winner      string // winner will help the winning md5sum
+	VotePerNode map[string]ConflictVote
+}
+
+type ConflictVote struct {
+	MD5   string
+	Error string
+}
+
+func (cs Conflicts) Merge(c Conflict) Conflicts {
```

Comment:
It looks that it makes sense to use pointer here and in all other methods.
```suggestion
func (cs *Conflicts) Merge(c Conflict) Conflicts {
```

Reply:
@artemgavrilov "Conflicts" is a slice of pointers, Conflicts []*Conflict, because they can be shared
Which is why I kept a value ref, as pointers to slice would not serve much as far as I know
(it would also force an iteration over "range *cs")